app/views/hours/_form.html.slim

```diff
@@ -4,7 +4,8 @@
   .row
     .col-xs-12
       - if action_name ==  'new'
-        = f.association :assignment, collection: @assignments_clients
+        = f.label :assignment
+        = f.select :assignment_id, options_from_collection_for_select(@assignments, 'id', 'client'), {}, class: 'form-control'
```

Comment:
Ah sorry forgot about simple_form :) Apparently you can also [do this](https://github.com/plataformatec/simple_form#associations):

```ruby
f.association :assignment, collection: @assignments_clients, label_method: :client, value_method: :id
```